README.markdown

```diff
@@ -72,6 +64,10 @@ _.mixin(s.exports());
 But it's not recommended since some of the are dropped because they collide
 with the functions already defined by Underscore.js.
 
+## Download
+
+  * [Development version](https://raw.github.com/epeli/underscore.string/master/dist/underscore.string.js) *Uncompressed with Comments*
+  * [Production version](https://github.com/epeli/underscore.string/raw/master/dist/underscore.string.min.js) *Minified*
```

Comment:
Its kept in sync with the repo FYI, but I agree it's not necessary

https://github.com/jsdelivr/jsdelivr/blob/master/files/underscore.string/update.json
On Jan 12, 2015 9:56 AM, "Esa-Matti Suuronen" <notifications@github.com>
wrote:

> In README.markdown
> <https://github.com/epeli/underscore.string/pull/378#discussion-diff-22794226>
> :
>
> > @@ -72,6 +64,10 @@ _.mixin(s.exports());
> >  But it's not recommended since some of the are dropped because they collide
> >  with the functions already defined by Underscore.js.
> >
> > +## Download
> > +
> > +  * [Development version](https://raw.github.com/epeli/underscore.string/master/dist/underscore.string.js) *Uncompressed with Comments*
> > +  * [Production version](https://github.com/epeli/underscore.string/raw/master/dist/underscore.string.min.js) *Minified*
>
> They are not maintained by us.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/epeli/underscore.string/pull/378/files#r22794226>.
>